source/speech/speech.py

```diff
@@ -3093,3 +3094,47 @@ def clearTypedWordBuffer() -> None:
 	complete the word (such as a focus change or choosing to move the caret).
 	"""
 	_curWordChars.clear()
+
+
+def getSpeechSequenceWithLangs(speechSequence: SpeechSequence):
```

Comment:
please move this outside of `speech.speech`, adding to `speech.speech` is discouraged. Please create a new submodule e.g. `speech.languageHandling` like `speech.speechWithoutPauses`